ci-operator/step-registry/rosa/e2e/ocm-fvt/rosa-e2e-ocm-fvt-commands.sh

```
@@ -75,4 +75,6 @@ echo "Running ocmtest: ${ocmtest_args[*]}"
 podman run \
   "${podman_args[@]}" \
   quay.io/redhat-services-prod/ocmci/ocmci:latest \
-  ocmtest "${ocmtest_args[@]}"
+  ocmtest "${ocmtest_args[@]}" 2>&1 | tee "${ARTIFACT_DIR}/ocmtest-output.log"
+exit_code=${PIPESTATUS[0]}
+exit $exit_code
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)